Resources/public/views/recommendations.html.twig

```diff
@@ -0,0 +1,31 @@
+<div id="recommendation-{{ recommendationId }}" class="ez-recommendation">
+    {{ 'Loading recommended items...'|trans }}
+</div>
+
+<script id="recommendation-template-{{ recommendationId }}" type="text/x-handlebars-template">
```

Comment:
as far as I understand, we include the same template as many times as we have recommendation blocks, we can do better I guess.